presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcIdentity.java

```diff
@@ -26,15 +26,17 @@
 {
     public static JdbcIdentity from(ConnectorSession session)
     {
-        return new JdbcIdentity(session.getIdentity().getUser(), session.getIdentity().getExtraCredentials());
+        return new JdbcIdentity(session.getIdentity().getUser(), session.getQueryId(), session.getIdentity().getExtraCredentials());
```

Comment:
JDBCIdentity is used to passing credential informations to make connections. Instead of adding queryId here. Can we add `ConnectorSession` to following method:
```
PeparedStatement buildSql(Connection connection, JdbcSplit split, List<JdbcColumnHandle> columnHandles)
```